pkgs/development/python-modules/buienradar/default.nix

```diff
@@ -33,7 +36,13 @@ buildPythonPackage rec {
     xmltodict
   ];
 
-  nativeCheckInputs = [ pytestCheckHook ];
+  nativeCheckInputs = [
+    pytestCheckHook
+    requests-mock
+    syrupy
+  ];
+
+  pytestFlagsArray = [ "--snapshot-update" ];
```

Comment:
That's like not running the syrupy tests.

Reply:
I'm open for suggestions how to deal with the 8 failing and 4 unused snapshots in another way.

Reply:
See https://github.com/NixOS/nixpkgs/pull/388084 for my suggestion.